apps/scraper-runner/src/main.ts

```diff
@@ -0,0 +1,79 @@
+import * as scholarsScrape from '@foodmedicine/scholars-scraper';
+import * as articleParser from '@foodmedicine/article-parser';
+import * as healthSiteScraper from '@foodmedicine/health-site-scraper';
+import * as fs from 'fs';
+import {
+  ParsedArticleParagraphStandalone,
+  ParsedArticle,
+  ParsedArticleParagraph,
+} from '@foodmedicine/interfaces';
+
+/**
+ * Get the top percentage of an array
+ * @param arr an array sorted in descending order
+ * @param percent percentage of items to be returned
```

Comment:
For readability's sake, can you add a dash or colon or something between the param name and description?
eg: `@param arr - an array sorted...`